ui/app/actions.js

```diff
@@ -264,6 +265,21 @@ function addNewKeyring (type, opts) {
   }
 }
 
+function importNewAccount (strategy, args) {
+  return (dispatch) => {
+    dispatch(actions.showLoadingIndication('This may take a while, be patient.'))
```

Comment:
Actually maybe we could know, we could parse the JSON file for the number of hashes... /shrug...